model/model.js

```diff
@@ -144,3 +143,36 @@ RETURNING *;
         });
     });
 };
+
+exports.editVotes = ({ review_id }, voteInc) => {
+  if (isNaN(voteInc)) {
+    return Promise.reject({ status: 400, msg: `Invalid data type` });
+  }
```

Comment:
this isn't needed.  If you try to make a query with an invalid data type Postgres will throw an error automatically which you can handle.

Reply:
but it does not